cloudomate/hoster/vps/vps_hoster.py

```diff
@@ -0,0 +1,99 @@
+from cloudomate import wallet as wallet_util
+
+from cloudomate.hoster.hoster import Hoster
+
+
+class VpsHoster(Hoster):
+    required_settings = None
+    configurations = None
+    client_area = None
+
+    def options(self):
+        """
+        Retrieve hosting options at Hoster.
+        :return: A list of hosting options
+        """
+        options = self.start()
+        self.configurations = list(options)
```

Comment:
This is a getter method with a side effect which is possibly not expected by developers. Can we somehow get rid of the assignment on line 17?